lib/stormpath-sdk/data_store.rb

```diff
@@ -348,6 +348,10 @@ def resource_is_application_web_config(resource, name)
       Stormpath::Resource::ApplicationWebConfig::ENDPOINTS.include?(name.underscore.to_sym)
   end
 
+  def user_info_mapping_rules?(result)
+    result['href'].include?('/userInfoMappingRules/') && result.key?('items')
+  end
+
```

Comment:
Couldn't you make basically the same method as `resource_is_saml_mapping_rules?` ?